### PR TITLE
[WIP] "Not enough comments" #607 patch

### LINKED ISSUE #607: Not enough comments
I've been told by my CS teacher that I should be commenting every line of code. I don't see that here, and that makes me worried.

### PR DESCRIPTION
**Pull request by dashmatic**
Add comments.
This will close #607.
Original conversation was at #608; moved due to internal technical difficulties.

### CODE REVIEW

File: src/test/java/FizzBuzzTest.java [hunk 1]
```diff
@@ -1,85 +1,133 @@
...
+		// set the system output to the returned object of the constructor for PrintStream imported from java.io.PrintStream on line 14 passing in bos declared on line 77
 		System.setOut(new PrintStream(bos));
 
+		// call the method fizzBuzz on the property fb of the current object, passing in the constant parameter n of type int
 		this.fb.fizzBuzz(n);
 
+		// flush the system ouput by running the method flush on the property out on the class System
 		System.out.flush();
+		// create a string variable called platformDependentExpectedResult which replaces all of the newlines in the constant parameter s of type string with the system line seperator, fetched by passing "line.seperator" to the method getProperty on class System
 		String platformDependentExpectedResult = s.replaceAll("\\n", System.getProperty("line.separator"));
```
Comment (dashmatic) [inline src/test/java/FizzBuzzTest.java:87]: Notes for anyone in the future; this line of code is currently not Enterprise Quality&trade; due to the fact that it does not import `com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.impl.Constants` and use the constant `LINE_SEPERATOR`. To make this Enterprise Quality&trade;, is it possible for somebody to start a pull request to change this singular line of code and make it follow industry standard by adding as many imports as possible to greatly optimize the code?

**Review by onien-12 — APPROVED** [review]
It will increase team efficiency

**Comment by EngineerRunner** [comment]
Please make sure this doesn't break compatibility with OS/2 and Windows NT 4.0, which are both used by our company.

**Comment by wanderrful** [comment]
Please schedule a meeting with all of us so that we can align on whether this direction is the right way to go. Also, please deliver us a high level overview of the trade-offs and benefits you considered before deciding to go in this direction with these code changes.

edit: Make sure to include latency considerations, as well, because we need to make sure we do this properly before we make any commitments.

**Comment by dvtate** [comment]
According to the company style guide, all single-line comments must start with a capital letter for maximal readability.  Additionally if the comment is a complete sentence it should end with a period.

Please update your PR accordingly. Thank you.

**Review by braindigitalis — APPROVED** [review]
🤣

File: src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/StandardFizzBuzz.java [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
+// import package com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.strategies.FizzBuzzSolutionStrategy
 import com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.strategies.FizzBuzzSolutionStrategy;
 
 /**
  * Standard FizzBuzz
  */
 @Service
 public class StandardFizzBuzz implements FizzBuzz {
-
+	// create a private contstant called _fizzBuzzSolutionStrategyFactory of type FizzBuzzSolutionStrategyFactory imported from com.seriouscompany.business.java.fizzbuzz.packagenamingpackage.interfaces.factories.FizzBuzzSolutionStrategyFactory on line 15
```
Comment (mikavilpas) [inline src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/StandardFizzBuzz.java:24]: Typo, should read "constant"

**Comment by heerhaan** [comment]
I like the idea but we should think about accessibility too. We can't just "add" comments with no regard to the languages the reader may or may not speak, can we add a translation feature to this?

I've already invited everyone involved in here to bi-daily meetings of an hour, there we can discuss our plan of action, track the progress, which languages to support and maybe even determine what a language even is in the first place.

**Comment by mikavilpas** [comment]
> I like the idea but we should think about accessibility too. We can't just "add" comments with no regard to the languages the reader may or may not speak, can we add a translation feature to this?
> 
> I've already invited everyone involved in here to bi-daily meetings of an hour, there we can discuss our plan of action, track the progress, which languages to support and maybe even determine what a language even is in the first place.

Could you please see if this is already proposed in https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition/pull/549 ?

**Pull request closed without merging**